pyglider/plotting.py

```diff
@@ -223,55 +363,78 @@ def grid_plots(fname, plottingyaml):
+           # locator = mdates.AutoDateLocator() 
+           # formatter = mdates.ConciseDateFormatter(locator)
+           # formatter.offset_formats = ['',
+           #                             '',
+           #                             '',]
+           # ax.xaxis.set_major_locator(locator)
+           # ax.xaxis.set_major_formatter(formatter)
 
             min, max = _autoclim(ds[k])
             if vmin is not None:
                 min = vmin
             if vmax is not None:
                 max = vmax
             # make time windows:
+            if ax == axs[0]:
+                vmax = 300
+                cmap.set_over("black") 
+                ax.set_ylim([300, 0])
+
+            vmax = 300
+            cmap.set_over("black") 
             # get good profiles.  i.e. those w data
             ind = np.where(np.sum(np.isfinite(ds[k].values), axis=0)>10)[0]
             _log.debug(ind)
             _log.debug(len(ds.time))
             if len(ind) > 1:
                 time = ds.time[ind[1:]] + np.diff(ds.time[ind]) / 2
+              #  print('tim1',time)
                 time = np.hstack((time[0] - (time[1]-time[0]) / 2, time))
+             #   print('tim2',time)
                 depth = ds.depth[:-1] - np.diff(ds.depth)
                 depth = np.hstack((depth, depth[-1] + np.diff(ds.depth)[-1]))
                 pc = ax.pcolormesh(time, depth, ds[k][:, ind],
-                    rasterized=True, vmin=min, vmax=max, cmap=cmap, shading='auto')
+                    rasterized=True, vmin=min, vmax=max,cmap=cmap, shading='auto')
                 ax.contour(ds.time[ind], ds.depth, ds.potential_density[:, ind], colors='0.5',
                        levels=np.arange(22, 28, 0.5)+1000, linewidths=0.5, alpha=0.7)
 
                 _log.debug(ds[k])
 
-                fig.colorbar(pc, ax=ax, extend='both', shrink=0.6)
-            ax.set_title(ds[k].attrs['long_name'] + ' [' +
-                     ds[k].attrs['units'] + ']', loc='left', fontsize=9)
+               # fig.colorbar(pc, ax=ax, extend='both', shrink=0.6)  #jpnote commented
+                cbar = fig.colorbar(pc, ax=ax, extend='both', shrink=0.6)
+                cbar.ax.set_ylabel(ds[k].attrs['long_name'] + ' [' +
+                                 ds[k].attrs['units'] + ']', fontsize=9)
+           # ax.set_title(ds[k].attrs['long_name'] + ' [' +
+                   #  ds[k].attrs['units'] + ']', loc='left', fontsize=9) #jpnote commented 
+            ax.set_title(ds[k].attrs['long_name'] + ' over Depth [m]', loc='left', fontsize=9)      
             t0 = ds.time[0]
             t1 = ds.time[-1]
             ax.set_xlim([t0, t1])
-            ax.set_ylim([depmax, 0])
-            if n == 0:
-                ax.set_ylabel('DEPTH [m]')
+           # ax.set_ylim([depmax, 0])
+            ax.set_ylim([300, 0]) #jp changed (or should I leave it as dep max? )
+           # if n == 0:
+            ax.set_ylabel('DEPTH [m]')
+         #   ax.set_xlabel('TIME (May - June)')
```

Comment:
I'd revert these changes...

Reply:
This should be for the new plots that Julia made of different variables vs. depth. She was cutting them off at 300m so you could see the near-surface more easily (since the deep stuff doesn't change much.)

Reply:
Done.